src/Symfony/Component/Translation/Extractor/Visitor/TransMethodVisitor.php

```diff
@@ -39,7 +39,7 @@ public function leaveNode(Node $node): ?Node
             return null;
         }
 
-        $name = (string) $node->name;
+        $name = $node->name instanceof Node\Name ? $node->name->getLast() : (string) $node->name;
```

Comment:
If I understand correctly, because I added 
```
$nameResolver = new NodeVisitor\NameResolver();
            $traverser->addVisitor($nameResolver);
```
now, the namespace is resolved and the `name` doesn't match `trans` or `t` anymore but the full name instead.

This change seems legit to get back the previous behavior.

But does it means that if someone write:
```
use function Foo\Bar\t;

t('foo');
```
we're considering `'foo`' as a translation key too ?

Best fix would check for the namespace too (now we have it), but it might be more complex.

Reply:
I did consider including the namespace, but that would be a BC break for any project that relies on any `t()` function being extracted. That should probably be deprecated first.

This PR only restores the original behaviour.

Reply:
Seems legit, and nobody reported an issue with the old behavior so far. So let's go with this!